packages/shared/src/services/ClaudeSessionServiceSimple.ts

```
@@ -0,0 +1,140 @@
+        status: "active",
+        createdBy: "desktop",
+        lastActivity: Date.now(),
+        createdAt: Date.now(),
+        userId: userId || "current-user",
+        metadata: {},
+        version: 1
+      };
+      
+      return sessionData;
+    }),
+
```

Biome (2.1.2)

[error] This generator function doesn't contain yield.



(lint/correctness/useYield)